SU-HSDO/suhumsci
# STN-255 Big Polish Work

# [STN-255](https://sparkbox.atlassian.net/browse/STN-255) READY FOR REVIEW

## Summary
For this big polish card I documented all the changes in the Jira card: https://sparkbox.atlassian.net/browse/STN-255. The focus of this card is to get as much polish work done as we can.

## Steps to Test
1. In the CLI, run npm run test to confirm all tests pass.
2. You can go to the [Econ] local site (http://economics.suhumsci.loc) to see the changes locally.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
